packages/checkbox/src/useCheckbox.ts

```diff
@@ -13,9 +13,9 @@ import { ICheckboxGroupInstance, ICheckboxProps } from './checkbox'
 export const useCheckboxGroup = () => {
   //todo: ELEMENT
   const ELEMENT = null
-  const elForm = inject<any>('elForm')
-  const elFormItem = inject<any>('elFormItem')
-  const checkboxGroup = inject<ICheckboxGroupInstance>('CheckboxGroup')
+  const elForm = inject<any>('elForm', {})
+  const elFormItem = inject<any>('elFormItem', {})
+  const checkboxGroup = inject<ICheckboxGroupInstance>('CheckboxGroup', {})
```

Comment:
I think default value is needed here since checkbox can be used without form/checkboxGroup. @zouhangwithsweet